src/client/app/containers/LineChartContainer.ts

```diff
@@ -54,73 +78,75 @@ function mapStateToProps(state: State) {
 					throw new Error('Unacceptable condition: readingsData.readings is undefined.');
 				}
 
-				const dataPoints: ChartPoint[] = _.values(readingsData.readings).map(compressedReadingToChartPoint);
+				// Create two arrays for the x and y values. Fill the array with the data from the compressed readings
+				const xData: string[] = [];
+				const yData: number[] = [];
+				const hoverText: string[] = [];
+				_.values(readingsData.readings).forEach(reading => {
+					const timeReading = moment(reading.startTimestamp);
+					xData.push(timeReading.format('YYYY-MM-DD HH:mm:ss'));
+					yData.push(reading.reading);
+					hoverText.push(`<b> ${timeReading.format('dddd, MMM DD, YYYY hh:mm a')} </b> <br> ${label}: ${reading.reading} kW`);
+				});
 
+				// This variable contains all the elements (x and y values, line type, etc.) assigned to the data parameter of the Plotly object
 				datasets.push({
-					label,
-					data: dataPoints,
-					fill: false,
-					borderColor: getGraphColor(label)
+					name: label,
+					x: xData,
+					y: yData,
+					text: hoverText,
+					hoverinfo: 'text',
+					type: 'scatter',
+					mode: 'lines',
+					line: {
+						shape: 'spline',
+						width: 3
+					},
+					marker: {color: getGraphColor(label)}
 				});
 			}
 		}
 	}
 
-	const options = {
-		animation: {
-			duration: 0
+	// Customize the layout of the plot
+	const layout: any = {
+		autozise: true,
+		// width: 700,
+		// height: 700,
+		// title: 'First Test',
+		showlegend: true,
+		legend: {
+			x: 0,
+			y: 1.1,
+			orientation: 'h',
 		},
-		elements: {
-			point: {
-				radius: 0
-			}
+		yaxis: {
+			title: 'kW',
+			showgrid: true,
```

Comment:
It is okay to set showgrip: true but that is the default. Testing shows grids are present without this line (or in xaxis). Given we don't include all the defaults, maybe we should remove this?